src/Components/AdvancedSearch/AdvancedSearch.stories.tsx

```diff
@@ -32,14 +30,7 @@ Base.args = {
         return;
     },
     adapter: new MockAdapter(),
-    allowedPropertyValueTypes: [
-        'string',
-        'boolean',
-        'float',
-        'integer',
-        'double',
-        'long'
-    ],
+    allowedPropertyValueTypes: queryAllowedPropertyValueTypes,
```

Comment:
since it's a constant, please make the name in all caps. QUERY_ALLOWED_.....

Reply:
I'm following convention of other consts in that file, ModelledPropertyBuilder.types.ts. I would prefer not to change all of them to all caps.